src/renderer/shared/ui-kit/CardStack/CardStack.tsx

```diff
@@ -73,6 +85,32 @@ const Trigger = ({ sticky, children }: TriggerProps) => {
 };
 
 const Content = ({ children }: PropsWithChildren) => {
+  const { open } = useContext(Context);
+  const [measureRef, { height }] = useMeasure();
```

Comment:
Let's use existing implementation of resize observer (useResizeObserver). New lib only for such a small case triggers me a bit